src/antelope/chains/evm/telos-evm-testnet/index.ts

```diff
@@ -88,10 +88,18 @@ export default class TelosEVMTestnet extends EVMChainSettings {
     }
 
     getStlosContractAddress() {
-        return '0xa9991e4daa44922d00a78b6d986cdf628d46c4dd';
+        return '0xa9991E4daA44922D00a78B6D986cDf628d46C4DD';
     }
 
     getWtlosContractAddress() {
         return '0xaE85Bf723A9e74d6c663dd226996AC1b8d075AA9';
     }
+
+    getImportantTokensIdList(): string[] {
+        return [
+            this.constructTokenId(TOKEN),
+            this.constructTokenId({ symbol: 'STLOS', address: this.getStlosContractAddress() } as EvmToken),
+            this.constructTokenId({ symbol: 'WTLOS', address: this.getWtlosContractAddress() } as EvmToken),
+        ];
+    }
```

Comment:
I think `telos-evm-testnet/index.ts` and `telos-evm/index.ts` have plenty in common. I usually rely on environment variables to change the mainnet/testnet differences, but I'm not sure this is possible in this project.

Is this something that could be addressed in the future?

Reply:
This is fine for now, both as a chain-agnostic approach and separation from native that's being handled with env vars, we can address it later.